Formula/enigma.rb

```diff
@@ -1,72 +1,69 @@
 class Enigma < Formula
   desc "Puzzle game inspired by Oxyd and Rock'n'Roll"
   homepage "https://www.nongnu.org/enigma/"
-  url "https://downloads.sourceforge.net/project/enigma-game/Release%201.21/enigma-1.21.tar.gz"
-  sha256 "d872cf067d8eb560d3bb1cb17245814bc56ac3953ae1f12e2229c8eb6f82ce01"
-  license "GPL-2.0"
-  revision 4
+  url "https://github.com/Enigma-Game/Enigma/releases/download/1.30/Enigma-1.30-src.tar.gz"
+  sha256 "ae64b91fbc2b10970071d0d78ed5b4ede9ee3868de2e6e9569546fc58437f8af"
+  license "GPL-2.0-or-later"
+  revision 1
 
   livecheck do
     url :stable
-    regex(%r{url=.*?/enigma[._-]v?(\d+(?:\.\d+)+)\.t}i)
+    # This uses the git tag to get version numbers.
+    # Stable releases should be tagged like 1.21.1 or 1.30
+    regex(/v?(\d+(?:\.\d+)+)$/i)
```

Comment:
This is the standard

Reply:
I put that comment in as a reminder to us Enigma developers that this would require us to continue to tag releases in exactly that format. If required, I could delete the comments and make sure it is mentioned in our release checklist, which is a better place for it anyway.

Reply:
Are you often changing versioning standards if you don't document it otherwise? Either way, yeah the release checklist seems like a better location.

Reply:
Removed comments